Dockerfile

```diff
@@ -1,18 +1,29 @@
-FROM ghcr.io/wepublish/node:16.1 as production
+FROM node:16-alpine3.17 as dependencies
+
+WORKDIR /dependencies
 
 RUN apk update
+RUN apk add python3 gcc g++ make --update-cache
 
-USER node
-RUN mkdir -p /home/node/wepublish
-WORKDIR /home/node/wepublish
+WORKDIR /dependencies
+COPY libs/api/prisma/schema.prisma ./libs/api/prisma/schema.prisma
+COPY .npmrc package.json package-lock.json ./
+RUN npm ci
 
-COPY --chown=node:node . .
 
-RUN rm -rf .env
-RUN npm ci
-RUN npm run build
+FROM node:16-alpine3.17 as builder
+WORKDIR /build
+COPY . .
+RUN rm -rf .env*
+COPY --from=dependencies /dependencies .
+RUN npm run build --prod
 
+
+FROM node:16-alpine3.17 as runtime
```

Comment:
Using `FROM gcr.io/distroless/nodejs:16` here would save 80MB

Reply:
> Using `FROM gcr.io/distroless/nodejs:16` here would save 80MB

```
#12 0.276 container_linux.go:380: starting container process caused: exec: "/bin/sh": stat /bin/sh: no such file or directory
------
executor failed running [/bin/sh -c npm ci]: exit code: 1
```

I get this on distroless so wont't go there atm.